api/provider/aws/common.go

```diff
@@ -69,6 +69,19 @@ func lookupDeployNameAndVersion(store tag.Store, deployID string) (string, strin
 	return deployName, deployVersion, nil
 }
 
+func lookupServiceIDFromServiceARN(store tag.Store, serviceARN string) (string, error) {
```

Comment:
We shouldn't do this pattern for services. It is only used for tasks and deploys since they don't have a logical mapping of Layer0 ID -> AWS ID.

Reply:
✅